src/annotator/util/shadow-root.js

```diff
@@ -45,5 +45,12 @@ export function createShadowRoot(container) {
 
   const shadowRoot = container.attachShadow({ mode: 'open' });
   loadStyles(shadowRoot);
+
+  // @ts-ignore The window doesn't know about the polyfill
+  const applyFocusVisible = window.applyFocusVisiblePolyfill;
```

Comment:
Ideally it would be nicer to import the `applyFocusVisiblePolyfill` function from the `focus-visible` package. However, it doesn't look like the function is exported :(

Can you add a comment here about which package this function comes from.

Reply:
I agree, it would have been better to add the function directly.

I added the comment.